go/ql/integration-tests/all-platforms/go/bazel-sample-1/force_sequential_test_execution

```diff
@@ -0,0 +1 @@
+# go get has been observed to sometimes fail when multiple tests try to simultaneously fetch the same package.
```

Comment:
Can you also have a non-comment line with, let's say
`goget`?
Otherwise, this gets assigned the default group, which (for backwards compatibility reasons) is `gradle`, which isn't the group you want to be in

Reply:
Done